starlette/requests.py

```diff
@@ -50,6 +50,21 @@ def query_params(self) -> QueryParams:
             self._query_params = QueryParams(query_string)
         return self._query_params
 
+    @property
+    def cookies(self) -> typing.Dict[str, str]:
+        if self._cookies is None:
+            self._cookies = {}
+            cookie_header = self.headers.get("cookie")
+            if cookie_header:
+                cookie = http.cookies.SimpleCookie()
+                cookie.load(cookie_header)
+                for key, morsel in cookie.items():
+                    self._cookies[key] = morsel.value
+        return self._cookies
+
+    def get_cookie(self, key: str) -> typing.Union[str, None]:
+        return self.cookies.get(key, None)
```

Comment:
Not sure we really need this method, since we have `request.cookies.get()` anyways.

Reply:
Ok, I’d suggest drop it *for now* since the request/response cookies interfaces aren’t at all symmetrical in any case.

At a later point we should then add a smart `response.cookies` dict-like interface, and start to bring the two in-line, except for immutable interfaces on the request.